extensions/throw-downloads-folder/src/throw.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from "react";
+import { useNavigation, showHUD } from "@raycast/api";
+import * as fs from "fs/promises";
+import * as path from "path";
+
+export default function ClearDownloads() {
+  const { pop } = useNavigation();
+
+  useEffect(() => {
+    clearDownloadsFolder()
+      .then(() => {
+        showHUD("🗑️ Downloads folder cleared");
+        pop();
+      })
+      .catch((error) => {
+        console.error("Sorry, error trying to catch the downloads folder", error);
+      });
+  }, []);
+
+  async function clearDownloadsFolder() {
+    const downloadsPath = path.join(process.env.HOME || "", "Downloads");
+    const files = await fs.readdir(downloadsPath);
+    for (const file of files) {
+      const filePath = path.join(downloadsPath, file);
+      await fs.rm(filePath, { recursive: true, force: true });
+    }
+  }
+
+  return null;
+}
```

Comment:
* I would add a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) before deleting files

* I would use the [trash command](https://developers.raycast.com/api-reference/utilities#trash) from @raycast/utils instead

* I would get the path from `os:homedir` as in the example in the above link